docs/README.md

```diff
@@ -0,0 +1,110 @@
+---
+orphan: true
+---
+
+# Building Documentation
+
+This directory contains the documentation source files for NVIDIA Dynamo.
+
+## Prerequisites
+
+- Python 3.11 or later
+- [uv](https://docs.astral.sh/uv/) package manager
+
+## Build Instructions
+
+### Option 1: Dedicated Docs Environment (Recommended)
```

Comment:
do we need to set this environment variable in the local flow?

```
ENV VISITS_COUNTING_SCRIPT="//assets.adobedtm.com/b92787824f2e0e9b68dc2e993f9bd995339fe417/satelliteLib-7ba51e58dc61bcb0e9311aadd02a0108ab24cc6c.js"
```

Reply:
I don't think its necessary -- that ENV var is only needed for the actual production build that gets published to docs.nvidia.com ( for web analytics purposes), local build is more for testing out changes to docs